### PR TITLE
[Starlink-ast] update to 9.2.7

### PR DESCRIPTION
**Pull request by veggiesaurus**
Version update to 9.2.7

- #### What does your PR fix?  
Updates to AST 9.2.7

- #### Which triplets are supported/not supported?
No change

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
Yes

### CODE REVIEW

File: versions/s-/starlink-ast.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "b2fc65fd5b045e25316b97c41bbf8759f9e45e7d",
+      "version-semver": "9.2.7",
+      "port-version": 0
+    },
+    {
+      "git-tree": "ffb74cc325d150caf277bf6fc6a8e84a69420986",
+      "version-semver": "9.2.6",
+      "port-version": 0
+    },
```
Comment (JonLiu1993) [inline versions/s-/starlink-ast.json:12]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
All features are tested successfully in the following triplet:
- x86-windows
- x64-windows
- x64-windows-static

File: ports/starlink-ast/vcpkg.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "starlink-ast",
-  "version-semver": "9.2.5",
+  "version-semver": "9.2.7",
```
Comment (strega-nil-ms) [inline ports/starlink-ast/vcpkg.json:3]:
```suggestion
  "version": "9.2.7",
```

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**